src/lib/config.js

```diff
@@ -17,6 +17,10 @@ module.exports = {
   API_CRI_CALLBACK: "/cri/callback",
   API_SESSION_INITIALISE: "/session/initialise",
   API_BUILD_PROVEN_USER_IDENTITY_DETAILS: "/user/proven-identity-details",
+  APP_STORE_URL_ANDROID:
+    process.env.APP_STORE_URL_ANDROID || "https://play.google.com/store/apps",
+  APP_STORE_URL_APPLE:
+    process.env.APP_STORE_URL_APPLE || "https://www.apple.com/uk/app-store/",
```

Comment:
Could these be the links for the GOV.UK ID Check app for now for testing: 
https://apps.apple.com/gb/app/gov-uk-id-check/id1629050566
https://play.google.com/store/apps/details?id=uk.gov.documentchecking